Server/controllers/healthController.js

```
@@ -3,49 +3,102 @@ import prisma from '../config/database.js';
+
+  try {
+    // Run all checks in parallel and collect results
+    const results = await Promise.allSettled([
+      invitationsCheck,
+      databaseCheck,
+      systemCheck,
+    ]);
+
+    const checksResults = results.reduce((acc, curr) => {
+      if (curr.status === 'fulfilled') {
+        return { ...acc, ...curr.value };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)